client/blocks/comment-detail/index.jsx

```diff
@@ -68,8 +69,10 @@ export class CommentDetail extends Component {
 	}
 
 	deleteCommentPermanently = () => {
-		const { commentId, deleteCommentPermanently } = this.props;
-		deleteCommentPermanently( commentId );
+		const { commentId, deleteCommentPermanently, translate } = this.props;
+		if ( window.confirm( translate( 'Delete this comment permanently?' ) ) ) {
```

Comment:
Let's check for window before we call it. This can make component testing easier